tests/unit/ListPeopleAndRoles.spec.ts

```diff
@@ -356,15 +356,15 @@ describe('List People And Roles component - SP registration', () => {
     expect(peoplesListItem1.querySelector('.name-column .name').textContent)
       .toBe('Crazy Cat Consulting Company')
     expect(peoplesListItem1.querySelector('.name-column .email').textContent)
-      .toBe('proprietor@example.com')
+      .toContain('proprietor@example.com')
```

Comment:
This fix the error occurred after `npm run lint`. It's the same as that in line 367.
![image](https://github.com/bcgov/business-create-ui/assets/60866283/e5333435-d598-4520-ad42-52e55b8aba65)

Reply:
Yes, this is fine. It's because the text is on a newline, so the exact match fails.